### ISSUE TITLE
Radio options are not accessible using a keyboard

### ISSUE THREAD

**Issue by imelda-guzman** [department-of-veterans-affairs/vets-design-system-documentation#562]
Hello from the VAS-UI team. We have a feature request for the VA radio component.
We currently have these versions installed

- component-library v2.5.8
- web-component v0.9.4

The latest release logs don't appear to have this feature.  Please let me know if you need more information regarding how we intend to use this component.

### What Happens
Using the Tab key to navigate to elements on the page skips focus on the radio group options.

### What I Expect
As a user I would expect to be able to navigate into the radio group with the Tab key. Once in the radio group, I would expect to either Tab through options with Tab or Arrow keys, then make my selection with either the Enter or Spacebar key.
